src/components/application_manager/src/commands/mobile/register_app_interface_request.cc

```diff
@@ -638,6 +638,8 @@ void RegisterAppInterfaceRequest::SendRegisterAppInterfaceResponseToMobile() {
   SendOnAppRegisteredNotificationToHMI(
       *(application.get()), resumption, need_restore_vr);
 
+  // Start PTU after successfull registration
+  GetPolicyHandler().AddApplication(application->policy_app_id());
```

Comment:
@VVeremjova 
I'm thinking of moving that to PolicyHandler::OnAppRegisteredOnMobile. What do you think?